docker_zmq_backend/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.8
+
+RUN mkdir /app
+
+WORKDIR /app
```

Comment:
you actually don't need these here, because you're mounting to /app/whatever docker will automatically create /app when it does the mounts. Also if you need to change the workdir you can do that in docker-compose or at runtime

Reply:
Specifying a working directory can make sense if you actually do some work in that directory in the dockerfile. But in this case you aren't doing any work there, so it's not really serving any purpose. 

Furthermore, there's nothing special about the /app directory. You could mount your python source literally anywhere in the filesystem and it would still work fine.